src/main/java/de/retest/recheck/persistence/ReportUploadMetadata.java

```diff
@@ -1,15 +1,13 @@
 package de.retest.recheck.persistence;
 
-import java.net.URI;
-import java.util.List;
-
 import lombok.Builder;
 import lombok.Getter;
 
 @Builder
 @Getter
 public class ReportUploadMetadata {
```

Comment:
The name doesn't fit now. Suggestion: Strip the Metadata or rename to something like `ReportUploadContainer`.